definitions/divorce/json/CaseEventToFields/CaseEventToFields-general-referral-nonprod.json

```diff
@@ -108,5 +108,137 @@
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "GeneralReferralPaymentType",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "GeneralReferralPayment1",
+    "PageLabel": "General referral payment",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "SolicitorFeeAccountNumber",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "MANDATORY",
+    "PageID": "GeneralReferralPayment1",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "GeneralReferralPaymentType=\"feeAccount\""
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "FeeAccountReference",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "GeneralReferralPayment1",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "GeneralReferralPaymentType=\"feeAccount\""
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "D8HelpWithFeesReferenceNumber",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "MANDATORY",
+    "PageID": "GeneralReferralPayment1",
+    "PageDisplayOrder": 2,
```

Comment:
I think this should be page 1.
"PageDisplayOrder" is the position of the page the field will be put in :).

Reply:
Updated. 
Im still not entirely sure what effect
    "PageFieldDisplayOrder":
    "PageDisplayOrder": 
    "PageColumnNumber":
and 
    "PageID":
Actually do. I get different answers for different people